style.css

```diff
@@ -68,12 +66,17 @@ section a {
     color: #20677B;
 }
 
-.no-fill-button:hover, .no-fill-button.active{
+.no-fill-button:hover, .no-fill-button.active {
     border-color: #20677B;
     background-color: #20677B;
     color: white;
 }
 
+div [class*=col-md-6] {
```

Comment:
looks weird to me .. is it for the display flex bug with bootstrap ? 

if yes, could you add a comment on top of it so it clear it up